pom.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>com.rayfowler</groupId>
+    <artifactId>ROTP</artifactId>
+    <version>1.1</version>
```

Comment:
https://www.mojohaus.org/templating-maven-plugin/ could be used to transfer `version` into code. I suggest special class for that like [`ProjectInfo`](https://github.com/andrew-hoffman/halfnes/blob/master/src/main/java-templates/com/grapeshot/halfnes/ProjectInfo.java) which contains constants from pom.